thesis-api/main/database/models.py

```diff
@@ -19,7 +19,15 @@ class TermModel(Base):
     __tablename__ = 'terms'
 
     id = Column(Integer, primary_key=True, nullable=False)
-    name = Column(String, unique=True, nullable=False)
+    name = Column(String, nullable=False)
```

Comment:
dodanie id w teorii ok, problem w tym że dla każdego słowa "name" wikipedia wygeneruje dokładnie jeden ten sam artykuł, np. jeżeli będziemy mieć słowo Meksyk to dostaniemy albo definicję państwa albo stolicy, więc nie ma sensu trzymać w bazie haseł z tym samym polem "name" (chyba że znajdziemy obejście tego problemu)

Reply:
Ten problem niestety i tak wystąpi (jeśli nie znajdziemy obejścia). Jeżeli będę przepuszczał tylko 1 słowo Meksyk, to jaką mamy gwarancję, że to będzie akurat ten Meksyk, który wyskoczy z API Wikipedii? Nie wiem w sumie co z tym zrobić.